jenkins/check-for-build.jenkinsfile

```diff
@@ -23,6 +23,8 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
+            H 1 * * * %INPUT_MANIFEST=2.18.0/opensearch-dashboards-2.18.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip;TEST_MANIFEST=2.18.0/opensearch-dashboards-2.18.0-test.yml;TEST_PLATFORM=linux;TEST_DISTRIBUTION=tar
+            H 1 * * * %INPUT_MANIFEST=2.17.1/opensearch-2.17.1.yml;TARGET_JOB_NAME=distribution-build-opensearch;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip;TEST_MANIFEST=2.17.1/opensearch-2.17.1-test.yml;TEST_PLATFORM=linux;TEST_DISTRIBUTION=tar
```

Comment:
Maybe we should update the mechanic to only include test for minor not patch releases.

Reply:
Right! Was also thinking if we should disable integration test failure notification by default. It is enabled by default as of now.